src/Arcus.Observability.Correlation/Arcus.Observability.Correlation.csproj

```diff
@@ -17,11 +17,11 @@
     <GenerateDocumentationFile>true</GenerateDocumentationFile>
   </PropertyGroup>
 
-  <ItemGroup Condition="'$(TargetFramework)' == 'netcoreapp3.1'">
+  <ItemGroup Condition="'$(TargetFramework)' != 'netstandard2.0'">
     <FrameworkReference Include="Microsoft.AspNetCore.App" />
   </ItemGroup>
```

Comment:
is the dependency to AspNetCore necessary here ?
The new assembly `Arcus.Obsservability.Telemetry.Core` has a dependency on this assembly (Correlation), so that means that it is still not possible to make use of the functionalities exposed by this library in projects that are hosted in containers that do not have the aspnet framework installed.  (See my other comment in the converstations tab).

Reply:
I don't think so as well

Reply:
Ah, yes, it was previously required bc of the correlation middleware, but that's moved.

Reply:
I also remember a previous issue that we fixed on all the repo's where we were supposed to add the AspNetCore dependency to be used when we required netcoreapp3.1 dependencies. I guess when I see the comments here, that that issue proposal was wrong.

Reply:
I don't think we have to do that indeed, we should only add ASP.NET when we need it.

Reply:
Indeed, I think that this has already been discussed some time ago:  only add AspNetCore framework as a dependency when it's required.  And, I think that this is already done for most projects ?